File: src/main/java/com/randomperson22/piggymodtest/npc/georgie/GeorgieNpcRenderer.java
package com.randomperson22.piggymodtest.npc.georgie;

import net.minecraft.client.renderer.entity.RenderManager;
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer;

public class GeorgieNpcRenderer extends GeoEntityRenderer<EntityGeorgieNpc> {
    public GeorgieNpcRenderer(RenderManager renderManager) {
        super(renderManager, new GeorgieNpcModel());
        this.shadowSize = 0.5f;
        // Initialize your animation processor here if needed
	}
}

File: src/main/kotlin/com/randomperson22/piggymodtest/npc/georgie/GeorgieNpcRenderer.kt
package com.randomperson22.piggymodtest.npc.georgie

import net.minecraft.client.renderer.entity.RenderManager
import software.bernie.geckolib3.renderers.geo.GeoEntityRenderer

class GeorgieNpcRenderer(renderManager: RenderManager?) :
    GeoEntityRenderer<EntityGeorgieNpc?>(renderManager, GeorgieNpcModel()) {
    init {
        this.shadowSize = 0.5f
        // Initialize your animation processor here if needed
    }
}